investd/sources/__init__.py

```diff
@@ -15,12 +16,10 @@
 ]
 
 
-def ingest_sources_as_df() -> pd.DataFrame:
-    df_tx = pd.DataFrame(columns=Transaction.__dataclass_fields__.keys())
-    for source_class in sources:
-        source = source_class()
-        df_tx = pd.concat(
-            [df_tx, pd.DataFrame([tx for tx in source.load_transactions()])]
-        )
+def ingest_all() -> pd.DataFrame:
+    df_tx = pd.DataFrame(
+        data=chain.from_iterable(Source().load_transactions() for Source in sources),
```

Comment:
nice chaining btw :)

Reply:
Since finding out about `chain` in AoC now I see where to use it so often! :laughing: